### DISCUSSION TITLE
[SOLVED] Read this if some classes work but others don't

### DISCUSSION THREAD

**Discussion by wkd-kapsule** [tailwindlabs/tailwindcss#13191] [Help] [answered]
The classes are recognized by Tailwind but somehow they don't apply any css. When using none-arbitrary values, I get an inconsistent behavior above 4. Any help would be greatly appreciated!

![image](https://github.com/tailwindlabs/tailwindcss/assets/77295465/b2145c60-43ec-4e98-85dd-2ffbb6c697cb)
![image](https://github.com/tailwindlabs/tailwindcss/assets/77295465/83e3fda8-5909-4a5e-89fb-991e75c563cc)
![image](https://github.com/tailwindlabs/tailwindcss/assets/77295465/531691f3-d209-47a5-bb19-a0ecd0d50886)

![image](https://github.com/tailwindlabs/tailwindcss/assets/77295465/fc80027d-4834-4e6a-b698-f796f88a65f4)

![image](https://github.com/tailwindlabs/tailwindcss/assets/77295465/131bdd6d-c324-4161-9f3e-96a9c4f6ea6a)

![image](https://github.com/tailwindlabs/tailwindcss/assets/77295465/3b4b5f22-bb02-4f26-85e8-5d98d8f0c855)

![image](https://github.com/tailwindlabs/tailwindcss/assets/77295465/836db343-67ed-4ee7-9ddb-ffcac4a9a868)

**Comment by wongjn** [comment, marked as answer]
Consider checking:
- Tailwind has compiled since you added the class(es).
- The `content` file globs sufficiently cover the file that the code appears in.
- [String concatenation/interpolation is not used to build the class name](https://tailwindcss.com/docs/content-configuration#dynamic-class-names). 
- The class name would resolve to a Tailwind class that Tailwind would generate a rule for.

Otherwise, consider providing a project that reproduces the unexpected behavior.

**Reply by wkd-kapsule** [reply]
Thanks you very much! The file was out of the `content` scope 😅. Still, I am curious to know why, on the same out-of-scope file, were some classes working and others weren't? For example `gap-8` was working but not `p-8`.

**Reply by wongjn** [reply]
Other classes would work if they exist in other files that *are* covered by the `content` file globs.

**Reply by wkd-kapsule** [reply]
Yeah I figured it out by digging more on how Tailwind builds the output css file. Thanks for your help
